tests/array_extensibility_test.py

```diff
@@ -29,6 +29,9 @@
 config.parse_flags_with_absl()
 
 
+@functools.partial(jax.tree_util.register_dataclass,
+                   data_fields=['x'],
```

Comment:
Do you need the `data_fields` part anymore?

Reply:
We need `data_fields` if the decorated class is not a `dataclass`